data/datapackage.yaml

```diff
@@ -0,0 +1,338 @@
+        - name: interventions
+          type: any
+          description: A URI to the linked annotation (if Derrida had a book where this reference was annotated.)
+          # ?: Is this accurate
+        - name: section
+          type: string
+          description: Section of De la grammatologie where the reference is located
+          constraints:
+            pattern: Part 1|Part 2
+          constraints:
+            required: true
+        - name: chapter
+          type: string
+          description: Chapter of De la grammatologie where the reference is located
+    scheme: file
+  - encoding: utf-8-sig
+    format: csv
+    hashing: md5
+    name: library
+    path: library.csv
+    description: Books from Derrida's personal library
+    profile: tabular-data-resource
+    schema:
+      primaryKey: id
+      foreignKeys:
+        # Frictionless Data doesn't seem to run tests on self-referencing foreign keys.
+        #  Whether all of collected_work_uri exists in the id column was checked manually.
+        #  When introducing an error into the dataset, Frictionless Data could not detect it.
+        #  https://specs.frictionlessdata.io/table-schema/#foreign-keys
+        - fields: collected_work_uri
+          reference:
+            resource: ""
+            fields: id
+      fields:
+        - name: id
+          type: string
+          format: uri
+          description: Unique link to the book's detail page
+          constraints:
+            required: true
+            unique: true
+        - name: item_type
+          type: string
+          description: The type of work (either a book, journal article, or book section)
+          constraints:
+            pattern: Book|Journal Article|Book Section
+          constraints:
+            required: true
+        - name: title
+          type: string
+          description: Title of the given work
+          constraints:
+            required: true
+        - name: alternate_title
+          type: any
+          description: '' # This means something specific, right?
+        - name: work_year
+          type: integer
+          description: Year in which the work was published
+        - name: copyright_year
+          type: integer
+          description: Year in which the work was copyrighted
+        - name: print_date
+          type: any
+          description: Print date of the work
+        - name: authors
+          type: string
+          description: Authors of the work
+        - name: contributors
+          type: any
+          description: Known contributors to the work (e.g translators or editors)
+        - name: publisher
+          type: string
+          description: Publisher of the work
+        - name: pub_place
+          type: string
+          description: City in which the work was published
+        - name: is_extant
+          type: boolean
+          description: Work exists in Derrida's library # ?
```

Comment:
pretty sure this means that PUL has a copy

Reply:
Whoops, just checked the help text: "Extant in Derrida's Library"